app/helpers/admin/holds_helper.rb

```diff
@@ -0,0 +1,13 @@
+module Admin
+  module HoldsHelper
+    def wait_time(hold)
```

Comment:
FWIW I don't fully understand the meaning of this column in the table. I see it's the same as it was so NBD. 🙂

Reply:
I had to look at it for a while to figure out what it's doing. I'll add some comments to the branches for now, but I'm curious what staff would say about these values and whether they're useful or not.